Section 3.md

```diff
@@ -81,6 +81,12 @@ Even though Terraphistus' catalogue has been stagnant for almost 50 years, it is
 
 A colonial start-up, O'Neill Manufacturing meets the demands of a growing community of spacers looking for more out of their EVA suits. Space suits have been an essential part of space exploration, for obvious reasons. Over time, suits have improved from the clunky affairs of early space travel, and O'Niell can provide any kind of suit for any kind of job. Voidsuits are their most common product, though some Softsuits are still sold for emergency use. Recently though, O'Niell has released a line of affordable industrial hardsuits to complement their line of exosuits. While many of their products are considered middle of the line goods, operators of O'Niell's line of equipment swear by its reliability.
 
+- **Xion Industrial**
+
+Xion was a company founded long ago, originally producing industrial equipment and vehicles, from drills to mechs. It only came to become the company we know it as today during the robotics boom of the 24th century, constructing drones for many industrial applications. They expanded at an incredibly rapid pace, quickly becoming a well-known name across Sol, and being a major contributor to the replacement of many human workers by robots.
+
+These days, they�re much less powerful than they were then, as other, stronger corporations progressed forward, using newer innovations such as positronics, whilst Xion sits back and continues to mass-produce cheap drones. In more recent years, they�ve introduced a line of prosthetics with a heavy focus on function over form, again being primarily for industrial work.
```

Comment:
"These days, they're a far less powerful corporation then they were back then. Rather than moving forward with the newer innovations such as positronic technology, Xion continues to mass-produce their cheap drones. In more recent years, they've introduced a new line of prosthetic equipment who's primary focus is on function over design in the industrial workforce."